src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Repositories/ContributorTeamRepository.cs

```diff
@@ -84,7 +87,11 @@ public IQueryable<ContributorTeam> FindWhere(System.Linq.Expressions.Expression<
         /// <param name="newEntity">The contributor team for adding.</param>
         public void Add(ContributorTeam newEntity)
         {
-            throw new NotImplementedException();
+            var newUser = new ContributorTeamEntity();
+            _unitOfWork.Context.ContributorTeams.Add(newUser);
+            _unitOfWork.Commit();
+            newEntity.Id = newUser.Id;
+            //TODO: make unit test
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1135)